src/game.h

```diff
@@ -196,6 +196,9 @@ class game
          */
         void on_options_changed();
 
+        std::map<m_flag, int> m_flag_usage_stats;
```

Comment:
Does this really have to be in `game` (which is already large)? It would probably better fit into `MonsterGenerator`.